ONSdigital/rm-actionexporter-service
# SUS008 Social sample file pilot changes

### NOTE: This will need to be merged at the same time as the other sample file pilot change branches

# Motivation and Context

The format of social sample file is changing for the LMS pilot and the requirements for the print files have changed. This PR includes updates to the print files and new database columns they require.

# What has changed



* Updated social pre-notification and notification print file templates
* Added database columns for return by date and country as these now need to be persisted to be available for the print templates
* Removed address lines 3 and 4 from the address database table and java model class (These were only added for the social test 1 sample file format)
* Updated tests

# How to test?



* Build and run this and all other social-sample-file-pilot-changes branches of java services (links below)
* Run the social test setup in rm-tools with a pilot format sample file
* Check the two files are generated and correct to your sample file
* Run the acceptance tests

# Links



https://trello.com/c/PhK3grfe/208-sus008-add-new-fields-to-pre-notification-print-file-5
https://trello.com/c/E9gchpRl/209-add-new-fields-to-invitation-print-file-5
https://trello.com/c/3PK5xIDM/238-sample-file-changes-for-operational-test-8

https://github.com/ONSdigital/rm-samplesvc-api/pull/17
https://github.com/ONSdigital/rm-action-service/pull/74
https://github.com/ONSdigital/rm-actionsvc-api/pull/28
https://github.com/ONSdigital/rm-sample-service/pull/72
https://github.com/ONSdigital/rasrm-acceptance-tests/pull/166
